packages/material-ui-system/src/styled.test.js

```diff
@@ -288,7 +288,6 @@ describe('styled', () => {
         shouldForwardProp: (prop) => prop !== 'variant' && prop !== 'size' && prop !== 'sx',
         name: 'MuiTest',
         slot: 'Slot',
```

Comment:
```suggestion
        slot: 'Slot',
        overridesResolver: (props, styles) => styles.slot,
```

Revert unrelated changes